composer.json

```diff
@@ -11,10 +11,10 @@
     }
   ],  
   "require": {
-      "php": ">=7.0.0",
-      "behat/behat": "^3.0",
-      "phpunit/phpunit": ">=6.0 <8.0",
-      "symfony/dependency-injection": "^3.0"
+      "php": ">=7.4.0",
+      "phpunit/phpunit": "^9.5.0",
+      "behat/behat": "^3.12.0",

```

Comment:
Why do we need to increase this to behat 3.12? I prefer to make these version constrains as permissive as possible to give the maximum compatablity to anyone else who may want to use this public project.